cijoe/scripts/bench_plotter.py

```diff
@@ -83,22 +82,34 @@ def plot_attributes_from_step(step):
     }
 
 
-def draw_bar_plot(data, plot_attributes, xlabel=None, y_limit=None):
+def draw_bar_plot(data, plot_attributes, xlabel=None, ylabel="iops", y_limit=None):
```

Comment:
I think we should decide on a convention for the x/y-labels. 
Either they should both include a default string or both default to `None`.

Reply:
I have set both to None, and updated the `latency_plotter` to have same format, such that when importing additional scripts in CIJOE is supported, we can easily extract this function into a shared imported script.